_posts/2020-09-03-final-report-hugo.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title:  "GsoC Report: Trajectory New Generation: the trajectory format for the future of simulation"
+date:   2020-09-03 08:58:08 +1000
+categories: 
+---
+
+# This is the end?
+
+Well well well, here we are at the end of GSOC! What a ride it has been, giving
```

Comment:
```suggestion
Well well well, here we are at the end of GSoC! What a ride it has been, giving
```

Reply:
Should be consistent throughout... so if you go with GSoC then change the other GSOC -> GSoC occurencesm, too.